live-examples/css-examples/cursor/cursor.html

```diff
@@ -0,0 +1,257 @@
+<section id="example-choice-list" class="example-choice-list" data-property="cursor">
+  <div>
+  <div class="example-choice" initial-choice="true">
+    <pre><code class="language-css">cursor: auto;</code></pre>
+    <button type="button" class="copy hidden" aria-hidden="true">
+      <span class="visually-hidden">Copy to Clipboard</span>
+    </button>
+  </div>
```

Comment:
We should restrict the number of choices to 6, to avoid scrolling. Which 6 values you pick is up to you, but try to choose a representative/commonly used set. I know it's hard in a case like this when there are so many values.